Sources/MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -306,10 +306,9 @@ extension CarPlayMapViewController: StyleManagerDelegate {
     }
     
     public func styleManager(_ styleManager: StyleManager, didApply style: Style) {
-        let styleURL = style.previewMapStyleURL
         let mapboxMapStyle = navigationMapView.mapView.mapboxMap.style
         if mapboxMapStyle.uri?.rawValue != style.mapStyleURL.absoluteString {
-            mapboxMapStyle.uri = StyleURI(url: styleURL)
+            mapboxMapStyle.uri = StyleURI(url: style.mapStyleURL)
```

Comment:
CarPlayMapViewController appears in the browsing and previewing activities. These activities should use `previewMapStyleURL`, not `mapStyleURL`, which is intended for the navigating activity. Although the default Navigation Day and Navigation Night styles no longer distinguish between previewing and turn-by-turn navigation with two sets of styles, it remains possible and possibly desirable for a developer to keep the two scenarios separate.

Reply:
Ah, I've completely missed that. Reverted to the previous state.